modules/EnsEMBL/Web/SpeciesDefs.pm

```diff
@@ -878,7 +888,12 @@ sub _parse {
   my $datasets = [];
   my $aliases  = $tree->{'MULTI'}{'ENSEMBL_SPECIES_URL_MAP'};
   my $labels    = $tree->{'MULTI'}{'TAXON_LABEL'};  
-  foreach my $prodname (@$SiteDefs::PRODUCTION_NAMES) {
+
+  ## Loop through all keys, not just PRODUCTION_NAMES (need for collection dbs)
+  foreach my $key (sort keys %$tree) {
+    next unless (defined $tree->{$key}{'SPECIES_PRODUCTION_NAME'}); # skip if not a species key
```

Comment:
I am not sure this makes sense since $tree->{$key} should bring back Species production name

Reply:
No, if you look further up the _parse method, you'll see that this tree has other keys - don't confuse $tree here with the tree created by ConfigPacker. See here: https://github.com/Ensembl/ensembl-webcode/blob/feature/collections/modules/EnsEMBL/Web/SpeciesDefs.pm#L802

Reply:
Hence you only want the subhashes that contain actual species information. We could use any other relevant key to identify a species subhash, but SPECIES_PRODUCTION_NAME is mandatory and therefore most reliable.